src/pages/contributing/translation-program/contributors.js

```diff
@@ -0,0 +1,90 @@
+// Libraries
+import React from "react"
+import styled from "styled-components"
+
+// Components
+import Breadcrumbs from "../../../components/Breadcrumbs"
+import Link from "../../../components/Link"
+import Translation from "../../../components/Translation"
+import {
+  Content,
+  ListItem,
+  Page,
+} from "../../../components/SharedStyledComponents"
+
+// Data
+import allTimeData from "../../../data/translation-reports/alltime-data.json"
+
+// Styles
+const HorizontalUl = styled.ul`
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr 1fr 1fr;
+
+  @media (max-width: ${(props) => props.theme.breakpoints.xl}) {
+    grid-template-columns: 1fr 1fr 1fr 1fr;
+  }
+
+  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
+    grid-template-columns: 1fr 1fr 1fr;
+  }
+
+  @media (max-width: ${(props) => props.theme.breakpoints.m}) {
+    grid-template-columns: 1fr 1fr;
+  }
+`
+
+const Contributors = ({ location }) => {
+  return (
+    <Page>
```

Comment:
Shouldn't we have a `PageMetadata` here as well? I've loaded the page and I don't see a title in the browser tab.